services/worker-manager/src/providers/google.js

```diff
@@ -228,50 +228,40 @@ class GoogleProvider extends Provider {
         wt.providerData[this.providerId].running = wt.providerData[this.providerId].running || 0;
       });
     }
-    const regions = workerPool.config.regions;
 
     const toSpawn = await this.estimator.simple({
       workerPoolId,
       ...workerPool.config,
+      capacityPerInstance: 1, // TODO Hardcoded for now until estimator updates
```

Comment:
I think TODO's are OK, *when* they are connected to a bug, and link to that bug in the source.  Then if I find the TODO still in place, I can figure out whether it was done, or closed as WONTFIX, or remains to be done.

Reply:
This is the bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1579554

I'll mention it in my todo (in my PR) as well